### DISCUSSION TITLE
Platform-specific installs?

### DISCUSSION THREAD

**Discussion by jc00ke** [jdx/mise#1920] [Q&A]
Hi! First, thanks for `mise`, I'm a HUGE fan and I've appreciated the improvements from `asdf -> rtx -> mise`.

I have 2 Linux machines and a Mac, and one of the plugins I use is Linux-only, and I install it with the Cargo backend: `cargo:i3status-rs" = "latest"`. This fails on my Mac though because it's fundamentally not compatible, makes sense.

I looked through the [config](https://mise.jdx.dev/configuration.html) and [cargo backend](https://mise.jdx.dev/dev-tools/backends/cargo.html) docs and didn't find what I was looking for, but is there a way in the `config.toml` to specify the platform that a tool should be installed on? Or is there another trick you have to achieve the same result?

Maybe I need to use [profiles](https://mise.jdx.dev/profiles.html) or [templates](https://mise.jdx.dev/templates.html) and somehow no-op?

**Comment by jdx** [comment]
I'd use `MISE_ENV=linux` or something

**Reply by jc00ke** [reply]
Yeah, I thought about that. I might try to see if I can pull from `uname -s` or from other ENV vars, will report back.

**Comment by jc00ke** [comment]
I went with [an external script](https://github.com/jc00ke/dotfiles/commit/3b4ef1b717a17bc4c86e1f3c36f7dcb18835497c) that would return the version I wanted. Maybe there's a better way of doing it, but this works for now.
